### DISCUSSION TITLE
Mount SMB network folder inside docker HA addon

### DISCUSSION THREAD

**Discussion by obert0** [alexbelgium/hassio-addons#1224] [Q&A]
I am new in the ha addon/docker. I try to mount folder from my NAS to a docker container running TVHeadend (I use this repo as base: https://github.com/GauthamVarmaK/addon-tvheadend/tree/main). I use Jellyfin addon without issue and the drive map very well. So I reused the automodule section of alexbegium in the dockerfile to map my drive. The scripts start but I get this error in the mounting validation:
smb1 disabled -- no workgroup available
mount error 13 permission denied
Did you have any idea how I can succeed?

**Comment by alexbelgium** [comment]
Hi, have you enabled the permissions in config.yaml of your add-on?

**Reply by obert0** [reply]
Hi, thank you for helping me! 
I add those options into my config.yaml. I'm not sure everything is relevant, but I try. 
host_dbus: true
host_network: true
privileged: 
    - SYS_ADMIN
    - DAC_READ_SEARCH
    - NET_ADMIN

My first try was without those additions to the config and the error was "smb1 disabled -- no workgroup available", but the mount error was not there.
